glossary/kmem-dax/_definition

```diff
@@ -0,0 +1,6 @@
+**KMEM DAX** is a semi-transparent alternative to [Memory Mode](#memory-mode) for volatile use of PMem.
+[Device DAX](#device-dax) can be configured in [system-ram mode](https://pmem.io/ndctl/daxctl-reconfigure-device.html).
+This mode exposes PMem as a hotplugged memory region - KMEM DAX.
+Persistent Memory takes the form of a separate memory-only NUMA Node(s). In contrast to **Memory Mode**,
```

Comment:
Reading this now - it doesn't flow well.
`This mode exposes PMem as a hotplugged memory region. Configured this way, Persistent Memory takes the form of a separate memory-only NUMA Node(s).`